stackrox/stackrox
# ROX-25493: Add policy CRD to helm chart

This includes the `SecurityPolicy` CRD in the Helm chart.

* The CRD is copied from the config-controller directory with some helm templating boilerplate added.
* The CRD template file is added to the ignore file so that it is only installed with the ROX_POLICY_AS_CODE flag enabled.

Testing:

* Build roxctl
* Run `roxctl central generate openshift --image-defaults=opensource pvc` and see no CRD in the template
* Run `roxctl central generate openshift --image-defaults=opensource pvc` with `ROX_POLICY_AS_CODE=true` and see the CRD (and config-controller deployment) in the template.
